src/utils/getFilteredPeople.ts

```diff
@@ -0,0 +1,64 @@
+import { Person } from '../types';
+
+type Filters = {
+  sex: string | null;
+  query: string;
+  centuries: string[];
+  sort: string | null;
+  order: string | null;
```

Comment:
The `sort` and `order` parameters are typed as `string | null`, but they are defaulted to an empty string in the calling code. Ensure consistency by either allowing empty strings here or adjusting the calling code to use `null`.